plugin-mount/configuration.ui

```diff
@@ -19,17 +19,27 @@
      <property name="title">
       <string>Behaviour</string>
      </property>
-     <layout class="QHBoxLayout" name="horizontalLayout">
-      <item>
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="devAddedLabel">
         <property name="text">
          <string>When a device is connected:</string>
         </property>
        </widget>
       </item>
-      <item>
+      <item row="0" column="1" colspan="2">>
        <widget class="QComboBox" name="devAddedCombo"/>
       </item>
+      <item row="1" column="0" colspan="2">
+       <widget class="QLabel" name="ejectPressedLabel">
+        <property name="text">
+         <string>When eject button is pressed:</string>
```

Comment:
How about adding some explanation tooltip? For me just seeing this label, it wasn't obvious what button are we talking here about. And this even doesn't need to be a HW button... it can be configurable shortcut.

Reply:
Added a tooltip. While I agree with you that the shortcut can be configurable, implementing a mechanism similar to one used in the globalaccel module for a single shortcut seems a bit cumbersome. I hoped that the user could change the shortcut key in globalaccel module, but it is read-only there. Cannot think of a good option here.

Reply:
> I hoped that the user could change the shortcut key in globalaccel module, but it is read-only there

I don't get you here. The shortcut is perfectly modifiable in `lxqt-config-globalkeyshortcuts` and I did test this just by modifying it (as I don't have any "eject" hw button and I was lazy to fiddle with xdotool).

Reply:
When I wrote that, I only tried it with the previous version (I run LXQt on OpenSUSE Leap, so I had 0.12) and it did not work (no matter what shortcut I tried to set, it would just ignore it and proceed with the countdown). So I had the impression it was not customizable.

When I tried it again with the current version in Leap 15.2 (LXQt 0.14), changing the shortcut works until I logout. Afterwards, the shortcut fails to register and does not show up in `lxqt-config-globalkeyshortcuts` until I remove the respective entry from `~/.config/lxqt/globalkeyshortcuts.conf`. Then it works again after I logout, but if I customize it, it disappears in the next session. Seems like there is a bug here, but I can change the shortcut.